packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -635,30 +635,25 @@ export class DashboardModel {
         dashboards: UpdateMultipleDashboards[],
     ): Promise<Dashboard[]> {
         await this.database.transaction(async (trx) => {
-            try {
-                await Promise.all(
-                    dashboards.map(async (dashboard) => {
-                        const withSpaceId = dashboard.spaceUuid
-                            ? {
-                                  space_id: await getSpaceId(
-                                      this.database,
-                                      dashboard.spaceUuid,
-                                  ),
-                              }
-                            : {};
-                        await trx(DashboardsTableName)
-                            .update({
-                                name: dashboard.name,
-                                description: dashboard.description,
-                                ...withSpaceId,
-                            })
-                            .where('dashboard_uuid', dashboard.uuid);
-                    }),
-                );
-            } catch (e) {
-                trx.rollback(e);
```

Comment:
why don't we want those rollbacks ? are they applied automatically ?

Reply:
yes, knex handles that for us. This was a bad practice when we started using knex.

Reply:
That's right! About half way down this page: https://knexjs.org/guide/transactions.html

And in dev when the error happened it was inside this transaction block. And the knex threw another error (hiding the real one) saying "you're trying to exit a transaction that has already been closed" (or something like that).

This fixed the dev experience. It doesn't seem to happen in production though, which is curious